sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -723,7 +723,7 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
    * Strategy to convert [[TransformWithState]] logical operator to physical operator
    * in streaming plans.
    */
-  object TransformWithStateStrategy extends Strategy {
+  object TransformWithStateStreamingSrategy extends Strategy {
```

Comment:
nit: TransformWithStateStreamingSrategy -> TransformWithStateStreamingS`t`rategy

Reply:
Thanks for catching, refactored